cookbooks/proxyfs/files/default/usr/bin/unmount_and_stop_pfs

```diff
@@ -55,7 +55,10 @@ if [ -f /usr/bin/systemctl ]; then
     # sudo /usr/bin/systemctl stop winbind
     sudo /usr/bin/systemctl stop proxyfsd
     await_proxyfsd_shutdown
-    sudo /usr/bin/swift-init main stop
+
+    # NOTE: We purposely stop Swift with "swift-init all stop" and not
+    # "swift-init main stop".   See start_and_stop_pfs for more details.
+    sudo /usr/bin/swift-init all stop
     # sudo /usr/bin/systemctl stop swift
```

Comment:
Can I remove the above line "# sudo /usr/bin/systemctl stop swift" ?

Reply:
I think we expected to be able to use that line at some point, when we had properly set it up. Now I'm starting to think it won't happen in the near future, so yeah, get rid of it. Also, it's not that hard to re-write it. ;)